test/resources/examples/index.js

```diff
@@ -108,25 +108,64 @@ const shortRingBetterTrade = generateTestCase({
   ],
 })
 
-const fiveThouzy = new BN("5000")
-const tenThouzy = new BN("10000")
+const fiveThousand = new BN("5000")
```

Comment:
I like this change but it's not related to the PR title.

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2Fl1KVaj5UcbHwrBMqI%2Fgiphy.gif&token=sad" data-gitmeme-token="sad"><img src="https://media0.giphy.com/media/l1KVaj5UcbHwrBMqI/giphy.gif" title="Created by gitme.me with /sad"/></a>

Reply:
I am glad you like it. Will probably leave it in unless you have strong opinions about reverting it.

Reply:
No strong opinion, but please make it a habit of separating those out.